code/game/gamemodes/miniantags/revenant/revenant_abilities.dm

```diff
@@ -354,7 +377,7 @@
 				to_chat(user, "<span class='revennotice'>Their brain is too basic for our powers!</span>")
 				return FALSE
 			to_chat(user, "<span class='revenwarning'>We have seeded madness in [M]'s mind! It will continue to fester...</span>")
-			to_chat(M, "<span class='warning'>A horrible feeling decends upon you as your mind goes fuzzy...")
+			to_chat(M, "<span class='warning'>A horrible feeling decends upon you as your mind goes fuzzy... (FINISH THIS FUCKER)")
```

Comment:
I bet this is gonna make it into the game and some poor sod will be wondering which fucker they have to finish